getting_serious/puzzle_29.R

```diff
@@ -0,0 +1,47 @@
+## Ten people, two rooms, states are {0, 1, 2, ..., 10} (number of people in other room)
+states <- seq(0, 10)
+
+## Transition matrix P
+## P[i, j] is the probability of transitioning from state i to j
+P <- matrix(0, length(states), length(states))
+rownames(P) <- states
+colnames(P) <- states
+for(i in seq_along(states)[seq(2, length(states) - 1)]) {
```

Comment:
I would rather write `for (i in 2:(length(states) - 1))`. Why did you choose this more complicated expression?

Reply:
Good question!  I'll make it simpler on the next commit